p2p/node.go

```diff
@@ -294,16 +316,16 @@ func (n *QriNode) SendMessage(msg Message, replies chan Message, pids ...peer.ID
 			continue
 		}
 
-		s, err := n.Host.NewStream(n.Context(), peerID, QriProtocolID)
+		s, err := n.Host().NewStream(n.Context(), peerID, QriProtocolID)
 		if err != nil {
 			return fmt.Errorf("error opening stream: %s", err.Error())
 		}
 		defer s.Close()
 
 		// now that we have a confirmed working connection
 		// tag this peer as supporting the qri protocol in the connection manager
-		n.Host.ConnManager().TagPeer(peerID, qriConnManagerTag, qriConnManagerValue)
-		n.Host.Peerstore().AddAddr(peerID, s.Conn().RemoteMultiaddr(), pstore.TempAddrTTL)
+		n.Host().ConnManager().TagPeer(peerID, qriConnManagerTag, qriConnManagerValue)
```

Comment:
I think all of these should use the internal `n.host` instead of `n.Host()`, it'll save a function call.